_beats/testing/environments/snapshot.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.1'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.0.0-alpha1-SNAPSHOT
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.0.0-SNAPSHOT
```

Comment:
Is this going to be fixed also in beats, as it would be overwritten otherwise with the next update?

Reply:
Yes, that's already updated in beats master so we're just waiting on elastic/beats#9735 before doing a beats update.